### DISCUSSION TITLE
Naming Timeline Index files on saving

### DISCUSSION THREAD

**Discussion by tomwolsky** [CommandPost/CommandPost#2757] [General]
I would like to export multiple index files, for clips, tags, markers. If I can't combine them in a single file, it would be helpful if there was naming dialog on saving the file. Otherwise I have to put each into a separate folder. 

Thanks.

**Comment by latenitefilms** [comment]
Good suggestion @tomwolsky! The reason for this is we currently don't have any API for creating standard save dialog screens - which is why we don't have any in CommandPost. We really need to fix this. I've created a new issue here: #2758

**Comment by tomwolsky** [comment]
Thanks very much Chris.


All the best,

Tom

**Comment by DaniDitu** [comment]
-- Play forward & Backward
cp.apple.finalcutpro:launch()
--cp.apple.finalcutpro:doShortcut("JogForward"):Now() --Play forward (press L multiple times to increase the playback speed)
cp.apple.finalcutpro:doShortcut("JogBackward"):Now() --Play in reverse (press J multiple times to increase the reverse playback speed)
